swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/converter/property/SwaggerParamCollectionFormat.java

```diff
@@ -57,6 +68,41 @@ public String getSeparator() {
     if (null == rawParam) {
       return new String[0];
     }
-    return rawParam.split(separator);
+    return rawParam.split(separator, -1);
+  }
+
+  /**
+   * Join params with {@link #separator}.
+   * Null element will be ignored since {@code null} cannot be described in query array param.
+   *
+   * @return joined params, or return {@code null} if {@code params} is null or all elements of {@code params} are null.
+   */
+  public String joinParam(Collection<?> params) {
+    if (null == params) {
```

Comment:
try org.apache.commons.lang3.StringUtils#join(java.lang.Iterable<?>, char)?

Reply:
New implementation is committed.